project_work.py

```diff
@@ -0,0 +1,250 @@
+
+# initializing variables for each month to add to
+january = 0
+february = 0
+march = 0
+april = 0
+may = 0
+june = 0
+july = 0
+august = 0
+september = 0
+october = 0
+november = 0
+december = 0
+monthly_sales_dict = {}
+
+# adding each of the values by state into their respective variables
+for key, value in raw_contract_date_and_price_dict.items():
+    if value == "1":
+        january += int(key.replace(",", ""))
+    elif value == "2":
+        february += int(key.replace(",", ""))
+    elif value == "3":
+        march += int(key.replace(",", ""))
+    elif value == "4":
+        april += int(key.replace(",", ""))
+    elif value == "5":
+        may += int(key.replace(",", ""))
+    elif value == "6":
+        june += int(key.replace(",", ""))
+    elif value == "7":
+        july += int(key.replace(",", ""))
+    elif value == "8":
+        august += int(key.replace(",", ""))
+    elif value == "9":
+        september += int(key.replace(",", ""))
+    elif value == "10":
+        october += int(key.replace(",", ""))
+    elif value == "11":
+        november += int(key.replace(",", ""))
+    elif value == "12":
+        december += int(key.replace(",", ""))
+
+# creating a dictionary that holds revenue in each individual month and printing the max revenue and month
+contract_date_and_price_dict = {
+    "January": january,
+    "February": february,
+    "March": march,
+    "April": april,
+    "May": may,
+    "June": june,
+    "July": july,
+    "August": august,
+    "September": september,
+    "October": october,
+    "November": november,
+    "December": december
+}
+contract_date_and_price_dict_sorted = sorted(contract_date_and_price_dict.items(), key=lambda item: item[1], reverse=True)
+print(f"Maximum amount of sale ({contract_date_and_price_dict_sorted[0][1]}) was in {contract_date_and_price_dict_sorted[0][0]}.")
+
+print()
+print("=" * 32)
+print()
+print("=" * 8 + "Amount of sale based on car brands" + "=" * 8)
+print()
+
+# calculating percentages for different car brands and adding them to a dictionary
+brand_dict = {}
+for brand in brands_set:
+    brand_dict.update({brand: brands_list.count(brand)})
+others = 0
+percentage_brand_dict = {}
+other_category_cutoff = num_deals * 0.04
+for brand, sales in brand_dict.items():
+    if sales < other_category_cutoff:
+        others += sales / num_deals * 100
+    else:
+        percentage_brand_dict.update({brand: sales / num_deals * 100})
+percentage_brand_dict.update({"Others": others})
+percentage_brand_dict_sorted = sorted(percentage_brand_dict.items(), key=lambda item: item[1], reverse=True)
+for brand_and_percentage in percentage_brand_dict_sorted:
+    print(f"{brand_and_percentage[0]} : {round(brand_and_percentage[1], 2)}%")
+print()
+print("=" * 19)
+
+# data sets for entry into the graphs
+states = list(set(state_list))
+sales = list(state_dict.keys())
+
+
+
+# creates the bar chart for question 2
+fig = plt.figure(1)
+plt.bar(states, sales)

```

Comment:
```suggestion
plt.bar(states, sales, color= colors)

```
This will add colors to the bar graph so it isn't as boring and is cool like Brandon's graphs.